src/Components/InquiryArtwork/index.tsx

```diff
@@ -21,8 +25,8 @@ const OverlayBackground = styled.div`
   justify-content: center;
   align-items: center;
 `
-
-const Circle = styled.div`
+const Div: StyledFunction<CircleProps> = styled.div
+const Circle = Div`
```

Comment:
So I guess the `props` are no longer typed as `any` by default in this version of styled-components? :/

Reply:
Yeah I think that may have been around for awhile. You mean being able to use custom props without typing them, right?

Reply:
Yeah